### ISSUE TITLE
[Bug]: Could not open `File.stories.tsx` in the editor

### ISSUE THREAD

**Issue by ndelangen** [storybookjs/storybook#32591]
### Describe the bug

I get the following error in my terminal:
```
Could not open AccessibilityViolationChart.stories.tsx in the editor.
The editor process exited with an error: spawn cursor ENOENT ('cursor' command does not exist in 'PATH').
```

And a notification in the storybook UI:
```
Failed to open in editor
spawn cursor ENOENT ('cursor' command does not exist in 'PATH').
```

### Reproduction steps

When I installed `cursor` I did not install the command to my PATH.

I needed to follow this documentation to get it to work:
https://cursor.com/docs/cli/installation#post-installation-setup

### System

```bash
This started on storybook 10 beta
```

**Comment by ndelangen** [comment]
I opened this Issue, so this can be found via search.

I'll close the issue, and there's no action possible/required on storybook side.

**Comment by kerryj89** [comment]
I don't know if things have changed since this was last posted, but following https://cursor.com/docs/cli/installation#post-installation-setup wasn't enough for me even after sourcing shell and restarting Storybook.  I had to actually open command palette in Cursor and search `Shell Command: Install 'cursor' command` for it to register the `cursor` command.  After that it opened up the files fine 👍
